pyecharts/template.py

```diff
@@ -71,3 +67,4 @@ def ensure_echarts_is_in_the_front(dependencies):
 
 def online(host=constants.DEFAULT_HOST):
```

Comment:
add deprecation note here please. keep the function though. existing user are still using it so let's not break user's code.